src/images/openshot.qrc

```diff
@@ -320,5 +320,6 @@
         <file>Humanity/actions/16/locked.svg</file>
         <file>Humanity/actions/16/transform-move.png</file>
         <file>Humanity/actions/16/view-list-icons.png</file>
+        <file>Humanity/actions/custom/camera-photo-symbolic.svg</file>
```

Comment:
Ooh, I just noticed... if you're updating the `src/images/openshot.qrc` file, there's one more step necessary. That file is compiled into a binary format by a utility called `pyrcc` (Actually `pyrcc5` for PyQt5), which is what's read by OpenShot. So, as [`src/images/README`](https://github.com/OpenShot/openshot-qt/blob/develop/src/images/README) notes, you need to run the following command from that directory:
```sh
pyrcc5 openshot.qrc -o openshot_rc.py
```
And then you should check in the updates to `openshot_rc.py` as well.

Reply:
Thanks!  Done! 😁